nextcord/utils.py

```diff
@@ -1129,9 +1129,54 @@ def format_dt(dt: datetime.datetime, /, style: Optional[TimestampStyle] = None)
     """
     if not isinstance(dt, datetime.datetime):  # pyright: ignore[reportUnnecessaryIsInstance]
         raise InvalidArgument("'dt' must be of type 'datetime.datetime'")
+    return format_ts(int(dt.timestamp()), style=style)
```

Comment:
No need to re-int an integer
```suggestion
    return format_ts(dt.timestamp(), style=style)
```

Reply:
Its a float